webpack.config.js

```diff
@@ -2,6 +2,7 @@ const { resolve } = require('path');
 const HtmlWebPackPlugin = require('html-webpack-plugin');
 
 module.exports = {
+  entry: ['@babel/polyfill', './src/index.js'],
```

Comment:
We will need to replace it to corejs in future.
Please mark it and do not forget